service/src/main/java/bio/terra/drshub/tracking/TrackCall.java

```diff
@@ -0,0 +1,14 @@
+package bio.terra.drshub.tracking;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Add this annotation to a method to track its call as a user event in Bard. Note this should just
+ * be used within Controller classes
```

Comment:
Why should this be used within Controller classes only, is it to guard against multiple logs for the same root event?

Reply:
Yeah, I wanted to log the explicit controller call since it maps nicely to a URL and HTTP method which is ultimately what we're interested (no value in tracking internal method calls)

Reply:
Cool!  That could be helpful context to add in the docstring.

Reply:
fair enough!